linux-support.md

```diff
@@ -1,38 +1,41 @@
 # .NET Support and Compatibility for Linux Distributions
 
-.NET is [supported by Microsoft](os-lifecycle-policy.md) and by [various commercial vendors and the community](linux.md) on multiple Linux distributions.
+.NET is supported by multiple [commercial and community organizations](support.md) on [multiple Linux distributions](linux.md).
 
-.NET can typically be run on any Linux distribution, via:
+.NET can be run on any Linux distribution, via:
 
-- The Microsoft build, which is built to be broadly compatible.
+- Portable builds, which are built to be broadly compatible with most Linux distributions.
 - Distribution-specific builds, which are built specifically for a given distribution version (like Red Hat Enterprise Linux 9 or Ubuntu 22.04).
```

Comment:
should we mention this is done by other companies?

Reply:
The distribution-specific builds are not exclusively provided by other companies, but also the community. It's basically portable builds from MS and distro-specific ones from everyone else. The nod to RHEL and Ubuntu seems pretty strong already. Fair?